include/bluetooth/bluetooth.h

```diff
@@ -1714,6 +1714,15 @@ int bt_le_scan_stop(void);
  */
 void bt_le_scan_cb_register(struct bt_le_scan_cb *cb);
 
+/**
+ *  @brief Unregister scanner packet callbacks.
+ *
+ *  Remove the callback structure from the list of scanner callback.
+ *
+ *  @param cb Callback struct. Must point to static memory.
```

Comment:
I guess we don't have any requirement of this being static memory. It could be dynamically allocated as well. The important thing is that it's a pointer to a struct that was previously added with `bt_le_scan_cb_register()`.

Reply:
Done